static/modules/guielement.js

```diff
@@ -0,0 +1,26 @@
+export class GuiElement {
```

Comment:
It bugs me that GUI isn't capitalized, since it's an acronym, but I have no idea what the precedent is for this since there's stuff like [XMLHttpRequest](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) which makes no sense.

Reply:
I'd like to hear @bandaloo's opinion on "GUI" vs "Gui" in class names
I'm partial to "Gui" because camelcase is nice but if I'm outvoted I'm outvoted

Reply:
I've like Gui lowercase. even javascript functions are inconsistent like what joe pointed out